packages/type-utils/src/utils.ts

```diff
@@ -70,5 +70,5 @@ export type DefinedUnionMember<T> = T extends string ? T : never;
  */
 export const isArrayMember = <Value extends string, Subset extends Value>(
     value: Value,
-    arr: Subset[],
+    arr: readonly Subset[],
```

Comment:
- Problem was, that `readonly string[]` was not assignable to `string[]`.
- The function _should_ accept both.
- I believe this is the best way to place it
  - it does not _require_ `arr` to be `readonly`
  - it merely widens the acceptable input type
  - and the inferred generic type `Subset` is the same, because it's primitive value, so no such thing as readonly.